apps/src/sites/studio/pages/layouts/_age_interstitial.js

```diff
@@ -0,0 +1,57 @@
+import getScriptData from '@cdo/apps/util/getScriptData';
+
+function shouldSubmitBeEnabled() {
+  if ($("#usertype").find("select")[0].value === 'student') {
+    return $("#age").find("select")[0].value !== '';
+  } else {
+    return $("#email-address").find("input")[0].value !== '';
+  }
+}
+
+function refreshSubmitState() {
+  $("#edit_user").find(".btn").prop('disabled', !shouldSubmitBeEnabled());
+}
+
+$().ready(function () {
+  $("#age-modal").modal('show');
+  $("#email-address").hide();
+
+  $("#edit_user").find("select").on('change', function (event) {
+    refreshSubmitState();
+  });
+
+  $("#email-address").find("input").on('input', function (event) {
+    refreshSubmitState();
+  });
+
+  const sectionsAsStudent = getScriptData('ageInterstitial').hasSections;
+
+  if (sectionsAsStudent) {
+    $("#usertype").hide();
```

Comment:
There's enough state change going on here it really feels like this should be a React component.  I know this work is up against a deadline.  Does this need to ship right away, or is there time to build it the "right" way?